### PR TITLE
Update changelog for v1.13.3

### PR DESCRIPTION
**Pull request by alexsomesan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
 ## 1.13.3 (Unreleased)
 FEATURES:
+* Add support for readiness_gate on Pod spec (#949)
```
Comment (dak1n1) [inline CHANGELOG.md:3]: #949 is the PR. Usually we link to the issue https://github.com/hashicorp/terraform-provider-kubernetes/issues/811

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
+* Add Azure Managed disk to PV resource (#1011)
```
Comment (dak1n1) [inline CHANGELOG.md:4]: https://github.com/hashicorp/terraform-provider-kubernetes/issues/202

**Comment by alexsomesan** [comment]
Sure, I'll link the issues. I always tended to mention PRs since not all changes usually come with issues attached.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,13 @@
...
 
 BUG FIXES:
 
+* Fix annotation diffs on affinity tests (#993)
+* Fix api_group requirement in cluster_role_binding and role_binding (#1024)
+* Fix service test leaking ELBs (#1013)
```
Comment (dak1n1) [inline CHANGELOG.md:10]: https://github.com/hashicorp/terraform-provider-kubernetes/issues/947

**Pull request merged**